src/common/playlist/playlist-manager.ts

```diff
@@ -388,6 +424,32 @@ class PlaylistManager {
   public destroy(): void {
     this._eventManager.destroy();
   }
+
+  private prepareEntry(index: number): Promise<any> {
+    if (this._playlist.items[index].isPlayable()) return Promise.resolve({ sources: this._playlist.items[index].sources });
```

Comment:
are you using prettier?

Reply:
lint and prettier both, yes. they sometimes have conflicts. why, do you see an error here ?